docs/meta/PROJECT_GUIDE.md

```diff
@@ -74,3 +74,21 @@ To help Pipcook discover all the plugins, the project provides some rules to let
 Community plugins can also be submitted as built-in plugins through pull requests, but this requires nomination by a core collaborator and the approvals of at least 2 collaborators.
 
 [semver]: https://semver.org/
+
+### Nightly Release
+
+The Nightly release will be built and published to npm every midnight if there is any commit in the past day.
+The aim to have such a release is to provide a quick fix to existig bugs and access to new/experimental features that are not suitable to deploy under the production environment.
+
+#### Usage
+
+You can directly use the following command:
+```bash
+npm install @pipcook/pipcook-cli@nightly
```

Comment:
```suggestion
$ npm install @pipcook/pipcook-cli@nightly
```

Reply:
Oh, shall we use beta as the npm tag? See https://github.com/alibaba/pipcook/blob/master/packages/cli/src/actions/init.ts#L28.

Or we need to change it `pipcook init --nightly`? /cc @FeelyChau @rickycao-qy

Reply:
I see. I will use `beta` as the nightly release tag

Reply:
Remember updating documentation, either :p